From 7a50f7f305e268b610a42cabf503f1623d9404bb Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Tue, 21 Jan 2020 11:50:20 -0800
Subject: [PATCH] Given we don't run tests exclude dbusmock for now

This is causing issues with the build and it may be related to the
module not existing (uncertain given the meson error message:
ERROR: Function does not take positional arguments). Since we don't
run tests for meson projects at the moment though just skip the line
and reconsider it later on.
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 8d09149..dfb6a1f 100644
--- a/meson.build
+++ b/meson.build
@@ -126,7 +126,7 @@ if bluez_backend_enabled
   # Needed for the BlueZ tests
   pymod = import('python')
   py_installation = pymod.find_installation('python3')
-  python_dbusmock = py_installation.dependency('dbusmock')
+#  python_dbusmock = py_installation.dependency('dbusmock')
 endif
 
 if import_tool_enabled
-- 
2.25.0

